Wikipedia/Code/TableOfContentsCell.swift

```diff
@@ -11,6 +11,11 @@ class TableOfContentsCell: UITableViewCell {
         didSet {
             titleLabelTopConstraint.constant = titleIndentationLevel == 0 ? 19 : 11;
             indentationConstraint.constant =  indentationWidth * CGFloat(1 + titleIndentationLevel)
+            if (titleIndentationLevel == 0) {
+                accessibilityTraits = .header
+            } else {
+                accessibilityValue = String.localizedStringWithFormat(WMFLocalizedString("table-of-contents-subheading-label", value:"Subheading %1$d", comment:"VoiceOver label to indicate level of subheading in table of contents. %1$d is replaced by the level of subheading."), titleIndentationLevel)
```

Comment:
Is it possible to set `UIAccessibilityTextAttributeHeadingLevel` on the attributed string for the title or does that only work for input views?